game-core/src/test/java/games/strategy/triplea/delegate/battle/casualty/CasualtyOrderOfLossesTestOnBigWorldV3.java

```diff
@@ -165,4 +161,115 @@ void infantryAndArtillery() {
     assertThat(
         "Artillery has the better total power", result.get(1).getType(), is(testData.artillery));
   }
+
+  @Test
+  void nonAmphibiousMarineWithAmphibiousAssault() {
+    testData.addTech(new ImprovedArtillerySupportAdvance(testData.gameData));
+
+    final List<Unit> amphibUnits = new ArrayList<>();
+    amphibUnits.addAll(testData.tank(1));
+    amphibUnits.addAll(testData.artillery(1));
+    amphibUnits.addAll(testData.marine(1));
+
+    amphibUnits.forEach(
```

Comment:
Would it be just as well to do set the property for just the marine rather than all units? That would remove the forEach loop